entity-framework/core/modeling/custom-function-mapping.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Custom function mapping - EF Core
+description: Mapping user-defined functions to database functions
+author: maumar
+ms.date: 11/23/2020
+uid: core/modeling/custom-function-mapping
+---
+# Custom function mapping
```

Comment:
Should we call this Use defined function mapping?

Reply:
I like user-defined function mapping better, and it probably corresponds to the terminology most users know.

On the downside, there's the part about mapping a CLR method to arbitrary SQL expressions rather than to a database method, so UDF may work less well there.

Reply:
But custom function mapping is also ambiguous in that case. Functions are more server side thing (client side methods?).

Reply:
It is like there is no write name without writing a really long title to explain the massive feature we have. :trollface: 
I like the part of user-defined function mapping is that the "function mapping" is defined by user (and also mapping to user-defined function).

Reply:
Whatever we choose will most likely end up a bit ambiguous, at least to someone... We can pick one and go with it...

Reply:
I think my initial choice was user defined function also, but that was confusing because I associated UDF with the sql function specifically (https://docs.microsoft.com/en-us/sql/relational-databases/user-defined-functions/create-user-defined-functions-database-engine) and this feature is so much more. Also, custom function mapping was the name in the bug ;) I'm happy to change, don't mind too much